src/build/build.ts

```diff
@@ -38,8 +38,18 @@ export async function build(
     options: ProjectBuildOptions,
     polymerProject: PolymerProject): Promise<void> {
   const buildName = options.name || 'default';
-  const optimizeOptions:
-      OptimizeOptions = {css: options.css, js: options.js, html: options.html};
+
+  // If a config for the current buildName is present
+  // Merge the projects optimizeOptions with the options
+  // Commandline options will override the projects options
+  const projectOptions = polymerProject.config.builds.find(buildConfig => buildConfig.name === buildName);
```

Comment:
You can destructure css, js and html directly. Then line 49 instead of projectOptimizeOptions you can do `{css, js, html}`

Reply:
But what if one of the options is not available, then the option where it should be merged to will be overridden with `undefined`

Reply:
Also, haven't seen any regression tests for the build/cli options. So that's why I didn't add one